templates/InstallChocolatey.ps1.erb

```diff
@@ -21,7 +21,7 @@ $ErrorActionPreference = 'Stop'
 #try {
 
 # variables
-$url = '<%= @download_url %>'
+$url = '<%= @download_url %><% if @chocolatey_version.to_s =~ /\d+\./ %><%= @chocolatey_version %><% end %>'
```

Comment:
@ferventcoder - Right, this should work now: Download URI renders to ` https://chocolatey.org/api/v2/package/chocolatey/0.10.10`

Reply:
A catch would be if the last `/` is missing from `@download_url`. Should I add another regex? In this case, it would look nicer in it's own code block.

Reply:
The problem is you are only showing how to make it work with the community repo. What about Artifactory, Nexus, ProGet, Chocolatey Simple Server, a file location, or any other the other repository types?

90% of the time the expectation is that an organization is using Puppet and Chocolatey, which means ***not*** using the community package repository, even to install Chocolatey. See https://chocolatey.org/docs/how-to-setup-offline-installation for our guide on organizational use.

Reply:
I think all these use cases are fairly easy to cover; we only need to check if our download URL matches something like `/^http(s):\/\/.*api/v2/package.*\/$/`. IMHO the version param is only applicable when using a nuget server for downloads anyway.


> which means not using the community package repository

This is also our use case; however we do sync the choco package from community to or internal nuget and therefore I most likely end up installing the most recent version instead the version I specify in my manifest.